src/bci_build/package/appcontainers.py

```diff
@@ -130,6 +130,20 @@
 ]
 
 _MARIADB_ENTRYPOINT = (Path(__file__).parent / "mariadb" / "entrypoint.sh").read_bytes()
+_MARIADB_HEALTHCHECK = (Path(__file__).parent / "mariadb" / "healthcheck.sh").read_bytes()
+_MARIADB_GOSU = b"""#!/usr/bin/env bash
```

Comment:
why /usr/bin/env bash here when you hardcode /bin/bash later? you could use the same in both, and I think env is unnecessary here. we knwo what the location of bash is in SLE BCI containers